src/pages/Pools/index.tsx

```diff
@@ -87,6 +96,7 @@ function Title({
   sortBy,
 }: TitleProps) {
   const [openTokenModal, setOpenTokenModal] = useState(false)
+  const { t } = useTranslation()
```

Comment:
It makes sense to scope this if you like it keep translations clean

Reply:
I think we should first define how we scope translations, so everyone is aligned with the scoping strategy.